docs/gallery.yml

```
@@ -7,11 +7,18 @@ visualization-examples:
     path: examples/visualization/mpas_topology.ipynb
     thumbnail: _static/thumbnails/mpas_topology.png
 
+  - title: Plotting E3SMv2 Output with UXarray
+    path: examples/visualization/e3sm-load-viz.ipynb
+    thumbnail: _static/thumbnails/ts_e3sm_load.png
+
 workflow-examples:
   - title: Working with MPAS Grids
     path: examples/working-with-mpas-grids.ipynb
     thumbnail: _static/thumbnails/004-thumbnail.png
 
+  - title:  Calculations on E3SMv2 Output with UXarray
```

An idea for a more descriptive title, any thoughts? 

```suggestion
  - title:  Radiative Feedback Analysis on E3SMv2 Model Output
```